components/pictures/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {Fragment} from 'react';
+import axios from 'axios';
+import useSWR, {Key} from "swr";
+import {
+  ImageList,
+  ImageListItem,
+  Skeleton
+} from "@mui/material";
+import {Box} from '@mui/system';
+import PictureTile, {ApodType} from "../pictureTile";
+
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+export const fetcher = (url: string) => axios.get(url).then(res => res.data)
+
+
+function Pictures() {
+  const apiURL: Key = `https://api.nasa.gov/planetary/apod?start_date=${"2022-01-12"}&end_date=${"2022-01-26"}&thumbs=true&api_key=${process.env.NEXT_PUBLIC_NASA_API_KEY}`;
+  const {data: apods, error} = useSWR<ApodType[], boolean>(apiURL, fetcher)
+
+  const mobile = useMediaQuery('(max-width:600px)');
+  const xs = useMediaQuery('(min-width:600px)');
+  const sm = useMediaQuery('(min-width:900px)');
+  const md = useMediaQuery('(min-width:1200px)');
```

Comment:
Keep in mind the `DRY` rule - `Don't Repeat Yourself` - you can create a function which will return media query string given the number of pixels. Here there are only three repetitions, so it's not a big deal, but you can consider this